ext/js/display/display-anki.js

```diff
@@ -852,7 +857,26 @@ export class DisplayAnki {
         const noteIds = this._getNodeNoteIds(node);
         if (noteIds.length === 0) { return; }
         try {
-            await this._display.application.api.noteView(noteIds[0], this._noteGuiMode, false);
+            await this._display.application.api.viewNotes([noteIds[0]], this._noteGuiMode, false);
```

Comment:
This has to be `[noteIds[0]]` to only show the note the user clicks from the right-click duplicate notes context menu.

Reply:
Can you make `_viewNotes` just take a boolean param of like `multiple`, and if `multiple==false`, do something like `noteIds.slice(0, 1)`? The logic is otherwise the same I think.

Reply:
Done 👍 somewhat differently though:
All actions (entry from the right-click menu, using the hotkey) work fine with just passing `noteIds` without slicing.

Reply:
Also replaced the "viewNote" action with "viewNotes" and renamed several methods for clarity.